schema_based_on_diagram.sql

```diff
@@ -0,0 +1,13 @@
+-- database is created
+
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (id serial PRIMARY KEY, name VARCHAR(225), date_of_birth date);
+
+CREATE TABLE invoices (id serial PRIMARY KEY, total_amount DECIMAl, generated_at TIMESTAMP, payed_at TIMESTAMP, medical_history_id INT);
+
```

Comment:
- [ ] Kindly make sure to create the corresponding `JOIN` tables beside these ones that are not in the diagram as it's required on the requirements for this part of the project 💯